packages/now-client/src/utils/index.ts

```diff
@@ -1,6 +1,6 @@
 import { DeploymentFile } from './hashes';
 import { parse as parseUrl } from 'url';
-import { fetch as fetch_ } from 'fetch-h2';
+import fetch_ from 'node-fetch';
```

Comment:
Are the now-client tests passing locally?

Circle CI shows that this fails with `reason: Unexpected token B in JSON at position 0`